app/views/devise/mailer/email_changed_by_admin_notification.html.erb

```diff
@@ -0,0 +1,12 @@
+<p>Hello <%= @user.name %>,</p>
+
+<p>Someone has changed your <%= link_to 'GOV.UK', 'https://www.gov.uk' %> Signon <%= account_name %> email from <%= @email_was %> to <%= @user.email %>.</p>
```

Comment:
"Someone" sounds somewhat suspicious here.  Would something like "A signon administrator" be better to make it clear that it's an authorised person who's made the change?